docs/community_resources/release_process.md

```diff
@@ -4,13 +4,19 @@
 
 [[= product_name =]] has three distributions:
 
-- [[= product_name =]] is an open source Content Management System (CMS) developed by Ibexa together with the open source community. [[= product_name =]]'s code base is available on GitHub under the GPLv2 license. [[= product_name =]] comes with no commercial support and maintenance services. It is supported by the community on public channels.
-- [[= product_name_exp =]] is commercial software available under eZ Business User License (BUL) to [[= product_name_exp =]] subscribers. It is comprised of [[= product_name =]], additional enterprise support and maintenance services, as well as additional features which are not available in the open source software.
+- [[= product_name_content =]] is a multichannel and headless content management system.
+- [[= product_name_exp =]] is a modern modular Digital Experience Platform to build outstanding customer experiences
```

Comment:
```suggestion
- [[= product_name_exp =]] is a modern modular Digital Experience Platform for building outstanding customer experiences.
```

Reply:
This is actual phrasing from https://www.ibexa.co/products/ibexa-experience, copied here on purpose.

Reply:
Do you agree with the phrasing? (Something's off with it for me.) If you do, that's ok, let's just fix the fullstop.